tests/modules/test_serialization.cpp

```diff
@@ -19,66 +19,64 @@ std::vector<trtorch::CompileSpec::InputRange> toInputRangesDynamic(std::vector<s
 }
```

Comment:
can we add an aarch64 dla test for this now that we have aarch64 specific tests?

Reply:
Added a test case for DLA as well